server.js

```diff
@@ -1,16 +1,17 @@
 const express = require('express');
 const hex_sha1 = require('./sha1.js');
-const lunr = require('lunr');
 const fs = require('fs');
 const https = require('https');
 const http = require('http');
-import {ArrayFormatter, touchp, esc_query} from "./utils";
+import { ArrayFormatter, touchp, esc_query } from "./utils";
 const chokidar = require('chokidar');
 const Stream = require('stream');
 const Chain = require('stream-chain');
 const parser = require('stream-json');
 const StreamArray = require('stream-json/streamers/StreamArray');
 
+const { lunrIndexer, redisIndexer } = require('./search-index');
```

Comment:
These are two more imports I'm not sure where they come from...

Reply:
they come from the search-index plugin. I can change this to `import { lunrIndexer, redisIndexer } from "./search-index";`

Reply:
thats nicer ... but I should have seen it...